examples/leasing/leasing.cfg

```diff
@@ -0,0 +1,17 @@
+[hil]
+# Config file format (/etc/leasing.cfg):
+# node_list: names of the nodes
+# threshold: number of times that cron job should call the leasing
+# script before releasing the node
+# user_name:
+# password:
```

Comment:
Should have a description for the username & password fields (should mention specifcially that they're for a HIL project with admin rights).